firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -259,3 +272,72 @@ float getInstantaneousSlope(int radius, int xStepsRemaining, int yStepsRemaining
     
     return ((float) radius - xStepsRemaining) / yStepsRemaining;
 }
+
+// Moves upwards to the largest radius in full steps
+void moveToFirstCircle()
+{
+    setScale(yMotor, WHOLE_STEPS);  
+    digitalWrite(yMotor[DIR_PIN], UP);  
+    for (int i = 0; i < circleRadius[0]; i++)
+    {
+        // Y-Step
+        digitalWrite(yMotor[STEP_PIN], LOW);
+        digitalWrite(yMotor[STEP_PIN], HIGH);
+    }
+}
+
+void moveToNextCircle(int currentCircle)
+{
+    // Circle 0 starts at the top, circle 1 at the left, circle 2 at the bottom, continuing in a counterclockwise fashion
+    int quarter = currentCircle % 4;
+
+    // Both motors move in whole steps
+    setScale(xMotor, WHOLE_STEPS);  
+    setScale(yMotor, WHOLE_STEPS);
+
+    // Set Y-direction
+    if (quarter == QUARTER_TOP || quarter == QUARTER_LEFT)
+        digitalWrite(yMotor[DIR_PIN], DOWN);
+    else
+        digitalWrite(yMotor[DIR_PIN], UP);
+
+    // Set X-direction
+    if (quarter == QUARTER_TOP || quarter == QUARTER_RIGHT)
+        digitalWrite(xMotor[DIR_PIN], LEFT);
+    else
+        digitalWrite(xMotor[DIR_PIN], RIGHT);
+
+    // Step in both directions
+    for (int i = 0; i < circleRadius[currentCircle+1]; i++)
```

Comment:
Assuming this is the implementation we talked about on Thursday where you decompose the movement into a line with slope 1 followed by another line, I'd like you to elaborate in the comments

Reply:
I would suggest a comment above both explaining this idea, then have your comments before both steps explaining which step they're doing